phoenix/core/migrations/0032_auto_20190827_0728.py

```diff
@@ -0,0 +1,111 @@
+    for outage in Outage.objects.all():
+        eta = int(outage.eta)
+        outage.eta = map_eta(eta)
+        outage.save()
+        for outage_history in OutageHistory.objects.filter(outage_id=outage.id):
+            eta = int(outage_history.eta)
+            outage_history.eta = map_eta(eta)
+            outage_history.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("core", "0031_auto_20190612_1001")]
+
+    operations = [
+        migrations.RemoveField(model_name="outage", name="prompt_active"),
+        migrations.RemoveField(model_name="outage", name="prompt_for_eta_update"),
+        migrations.AddField(
+            model_name="outage",
+            name="lost_bookings_choice",
+            field=models.CharField(
+                choices=[
+                    ("0%", "0%"),
+                    ("<30%", "<30%"),
+                    ("<60%", "<60%"),
+                    ("<100%", "<100%"),
+                    ("100%", "100%"),
+                ],
+                default=0,
+                max_length=5,
+            ),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name="outagehistory",
+            name="lost_bookings_choice",
+            field=models.CharField(
+                choices=[
+                    ("0%", "0%"),
+                    ("<30%", "<30%"),
+                    ("<60%", "<60%"),
+                    ("<100%", "<100%"),
+                    ("100%", "100%"),
+                ],
+                default=0,
+                max_length=5,
+            ),
+            preserve_default=False,
+        ),
+        migrations.AlterField(
+            model_name="outage",
+            name="eta",
+            field=models.CharField(
+                choices=[
+                    ("<30m", "<30m"),
+                    ("<2h", "<2h"),
+                    ("<8h", "<8h"),
+                    ("<24h", "<24h"),
+                    (">24h", ">24h"),
+                ],
+                max_length=6,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="outage",
+            name="lost_bookings",
+            field=models.TextField(blank=True, max_length=3000, null=True),
+        ),
+        migrations.AlterField(
+            model_name="outagehistory",
+            name="eta",
+            field=models.CharField(
+                choices=[
+                    ("<30m", "<30m"),
+                    ("<2h", "<2h"),
+                    ("<8h", "<8h"),
+                    ("<24h", "<24h"),
+                    (">24h", ">24h"),
+                ],
+                max_length=6,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="outagehistory",
+            name="lost_bookings",
+            field=models.TextField(blank=True, max_length=3000, null=True),
+        ),
+        migrations.RunPython(transform_previous_eta_values),
```

Comment:
Did you try to run this migration locally on the DB with some data? I concerned, that some updates could run in the same transaction with ALTER TABLE

Reply:
Yes I tried it. This isn't first migration file with this kind of transformation either and there were no problems with this approach so far. 
I can split this and move the data transformation to separate migration file, it's not a problem.

Reply:
Ok, cool, I wasn't sure. Let's keep it as it is now